modules/metrics/src/main/java/com/spotify/apollo/metrics/semantic/MetricsConfig.java

```diff
@@ -42,6 +43,7 @@
       EnumSet.of(
           ENDPOINT_REQUEST_RATE,
           ENDPOINT_REQUEST_DURATION,
+          ENDPOINT_REQUEST_LATENCY_SLO_RATE,
```

Comment:
nitpick - should we just call everything a duration to remain consistent with the other metric names here? `ENDPOINT_REQUEST_DURATION_THRESHOLD_RATE`

Also I think this is very useful outside of the SLO use case since these duration metrics can actually be aggregated across instances. What do you think of a mass rename from `latency-slo-metrics` to `duration-threshold-metrics` or something similar?

Reply:
👍